.github/workflows/pytest.yml

```diff
@@ -27,8 +27,8 @@ jobs:
       - name: Install pip
         run: python -m pip install --upgrade pip
 
-      - name: Install pytest
-        run: pip install pytest
+      - name: Install packages
+        run: pip install pytest treelib
```

Comment:
I think we should add a poetry setup instead. (separate issue)

Reply:
Created an issue.